apps/web/src/components/templates/TriggerSnippetTabs.tsx

```diff
@@ -10,7 +10,7 @@ const novu = new Novu('<API_KEY>');
 
 novu.trigger('${trigger.identifier?.replace(/'/g, "\\'")}', {
   to: { 
-    subscriberId: '<REPLACE_WITH_USER_ID>', ${trigger.subscriberVariables
+    subscriberId: '<REPLACE_WITH_USER_ID>' ${trigger.subscriberVariables
```

Comment:
@Tosin-webdev there is a small edge case here:
If `trigger.subscriberVariables` is empty this will be great,
if `trigger.subscriberVariables` has values it will generate a bad snippet:
```
to: {
   subscriberId: '<REPLACE_WITH_USER_ID>' <----- Missing coma
   firstName: 'TEST',                      <---- Extra comma added
}
```

Reply:
So in case `trigger.subscriberVariables` have nothing we should render:
```
to: {
    subscriberId: '<REPLACE_WITH_USER_ID>'
}
```

Note there is no "," there.

And in case `trigger.subscriberVariables` has values:
```
to: {
    subscriberId: 'TEST',   <--- here it was added
    firstName: 'TESt' <--- and here removed
}
```

One thing we can do is to remove the hard coded subscriberId, and append it to an array so we will always do a Map, just it will always contain the subscriberId value in it so the Join method will work nicely. @Tosin-webdev what do you think?